addons/pos_restaurant/models/pos_config.py

```diff
@@ -63,3 +63,13 @@ def write(self, vals):
         if ('is_order_printer' in vals and vals['is_order_printer'] == False):
             vals['printer_ids'] = [(5, 0, 0)]
         return super(PosConfig, self).write(vals)
+
+    def get_all_table_draft_orders(self):
+        self.ensure_one()
+        self = self.with_context(prefetch_fields=False)
+        order_obj = self.env['pos.order']
```

Comment:
The point of that variable is not re usability, is performance:

![speed](https://user-images.githubusercontent.com/34751857/203397309-1412e977-bf6c-4960-9fea-e1528557680f.png)

Since this is a server side change, we expect a lot of request, so better be fast.

Reply:
Fair enough since it's indeed accessed multiple time